en/docs/manage-databases-and-caches/choreo-managed-postgresql-databases.md

```diff
@@ -1,21 +1,23 @@
-# Choreo-Managed PostgreSQL Databases
+# Choreo-Managed PostgreSQL Databases/Vector Databases
 
-PostgreSQL on Choreo offers fully Choreo-managed, efficient object-relational databases on AWS, Azure, GCP, and Digital Ocean.
+PostgreSQL on Choreo offers fully Choreo-managed, efficient object-relational databases on AWS, Azure, GCP, and Digital Ocean which also can be used as a Vector database. 
 
 ## Create a Choreo-managed PostgreSQL database
 
 Follow the steps below to create a Choreo-managed PostgreSQL database: 
 
 1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
 2. In the header, click the **Organization** list. This opens the organization home page.
-2. In the left navigation menu, click **Dependencies** and then **Databases**.
-3. Click **Create** and select **PostgreSQL** as the database type. Provide a display name for this server and follow the instructions.
-4. Select your preferred cloud provider from AWS, Azure, GCP, or Digital Ocean.
+3. In the left navigation menu, click **Dependencies**.
```

Comment:
How about we have a separate heading for the vector database just after the Create a Choreo-managed PostgreSQL database section? 

Create a Choreo-managed PostgreSQL vector database

Follow these steps:

1. Sign in to the Choreo Console at https://console.choreo.dev/.
2. In the header, click the Organization list. This opens the organization home page.
3. In the left navigation menu, click Dependencies and then click Vector Databases.
4. Follow the steps 4 onwards in the “Create a Choreo-managed PostgreSQL database” section. (We can make “Create a Choreo-managed PostgreSQL database” a link that goes to the appropriate step)